src/build123d/geometry.py

```diff
@@ -1376,149 +1421,80 @@ class Location:
         Extrinsic.ZYZ: gp_EulerSequence.gp_Extrinsic_ZYZ,
     }
 
-    @overload
-    def __init__(self):  # pragma: no cover
-        """Empty location with not rotation or translation with respect to the original location."""
-
-    @overload
-    def __init__(self, location: Location):  # pragma: no cover
-        """Location with another given location."""
-
-    @overload
-    def __init__(self, translation: VectorLike, angle: float = 0):  # pragma: no cover
-        """Location with translation with respect to the original location.
-        If angle != 0 then the location includes a rotation around z-axis by angle"""
-
-    @overload
-    def __init__(
-        self, translation: VectorLike, rotation: RotationLike | None = None
-    ):  # pragma: no cover
-        """Location with translation with respect to the original location.
-        If rotation is not None then the location includes the rotation (see also Rotation class)
-        """
-
     @overload
     def __init__(
         self,
-        translation: VectorLike,
-        rotation: RotationLike,
-        ordering: Extrinsic | Intrinsic,
-    ):  # pragma: no cover
-        """Location with translation with respect to the original location.
-        If rotation is not None then the location includes the rotation (see also Rotation class)
-        ordering defaults to Intrinsic.XYZ, but can also be set to Extrinsic
+        position: VectorLike | None = None,
+        orientation: VectorLike | None = None,
+        ordering: Extrinsic | Intrinsic | None = None,
+    ):
+        """Location with position that defaults to the global origin.
+        If orientation is not None then the location includes the orientation
+        (see also Rotation class). Ordering defaults to Intrinsic.XYZ,
+        but can also be set to members of the Extrinsic enum.
         """
 
     @overload
-    def __init__(self, plane: Plane):  # pragma: no cover
+    def __init__(self, plane: Plane):
```

Comment:
The `# pragma: no cover` was to instruct the coverage calculator to ignore these lines as even if they were covered in the tests they are reported as not covered.  Has this changed now?